forge/src/main/java/gjum/minecraft/civ/snitchmod/forge/mixins/LevelRenderMixin.java

```diff
@@ -0,0 +1,21 @@
+package gjum.minecraft.civ.snitchmod.forge.mixins;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import gjum.minecraft.civ.snitchmod.forge.ForgeSnitchMod;
+import net.minecraft.client.renderer.LevelRenderer;
+import net.minecraft.client.renderer.RenderType;
+import org.joml.Matrix4f;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(LevelRenderer.class)
+public abstract class LevelRenderMixin { // REMINDER: Forge sucks
```

Comment:
would it make sense to use this mixin with all mod loaders? to reduce disparities which may lead to different behavior on different loaders

Reply:
We could but the mixin is currently just replicating the behaviour provided by the other loaders because Forge removed the event without a replacement.